servers/src/main/java/tachyon/master/file/FileSystemMaster.java

```diff
@@ -133,6 +133,11 @@
    */
   @SuppressFBWarnings("URF_UNREAD_FIELD")
   private Future<?> mTTLCheckerService;
+  /**
+   * The service that detects lost files. We store it here so that it can be accessed from tests.
+   */
+  @SuppressFBWarnings("URF_UNREAD_FIELD")
+  private Future<?> mLostFilesDetectionService;
```

Comment:
Does this mean we have to add another thread to the master, or was there an unused thread?